policy/modules/services/apache.fc

```diff
@@ -49,6 +49,13 @@ HOME_DIR/((www)|(web)|(public_html))(/.*)?/logs(/.*)?		gen_context(system_u:obje
 /usr/bin/httpd(\.worker)?				--	gen_context(system_u:object_r:httpd_exec_t,s0)
 /usr/bin/lighttpd					--	gen_context(system_u:object_r:httpd_exec_t,s0)
 /usr/bin/mongrel_rails					--	gen_context(system_u:object_r:httpd_exec_t,s0)
+
+ifdef(`distro_gentoo',`
+/usr/bin/nginx						--	gen_context(system_u:object_r:httpd_exec_t,s0)
+/usr/bin/php.*-fpm					--	gen_context(system_u:object_r:httpd_exec_t,s0)
+/usr/bin/php-fpm[^/]+					--	gen_context(system_u:object_r:httpd_exec_t,s0)
+')
+
```

Comment:
The nginx one doesn't need to be in the block; I'm unsure if the php ones should be inside. If the ifdef remains, it should move down after the other `/usr/bin` lines, at line 63.

Reply:
The general guideline is: if the full path is standard, then it doesn't need an `ifdef`. Refpolicy generally considers `/bin`, `/sbin`, `/usr/bin`, and `/usr/sbin` as equivalent and `nginx` is standard too, so I don't think it should be in an ifdef.

I'm not familiar with php, so I don't know if that file naming is standard or Gentoo-specific to handle multiple versions of php (or for some other reason).

Reply:
Ok, thanks for the explanation. So it’s not related to the progress of usrmerge in each distro.
This file-naming seems common pratice : [openSUSE](https://opensuse.pkgs.org/15.6/opensuse-oss-x86_64/php8-fpm-8.2.10-150600.1.6.x86_64.rpm.html)/[Fedora](https://rpmfind.net/linux/RPM/remi/enterprise/9/modular/x86_64/php-fpm-8.3.12-1.el9.remi.x86_64.html)/[Debian](https://packages.debian.org/bookworm/amd64/php8.2-fpm/filelist)